portfolio/src/main/webapp/script.js

```diff
@@ -71,10 +71,6 @@ function contact() {
 	console.log('Contact!'); 
 }
 
-function updateNumber() {
-    getComments();
-}
-
 function getComments() {
 	fetch('/data?max-num=' + (document.getElementById("numComments").value)).then(response => response.json()).then((comments) => {
```

Comment:
Maybe consider restructuring this line like this:
```
fetch('/data?max-num=' + (document.getElementById("numComments").value))
  .then(response => response.json())
  .then((comments) => {
```
so that you don't have an overly long line.

Reply:
thanks! updated!